test/integration/podman_test.go

```diff
@@ -6,11 +6,24 @@ import (
 	"runtime"
 
 	"github.com/crc-org/crc/v2/test/extended/crc/cmd"
+	"github.com/crc-org/crc/v2/test/extended/util"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 )
 
-var _ = Describe("podman preset", Serial, Label("podman-preset"), func() {
+var _ = Describe("podman preset", Serial, Ordered, Label("podman-preset"), func() {
```

Comment:
Is it intentional that the `Ordered` addition is in this `integration: cleanup to AfterAll and BeforeSuite` commit and not in `integration: run ginkgo nodes serially and in ordered fashion`? If it's not intentional, I've already split it in https://github.com/cfergeau/crc/commits/ginkgo-issue